skein/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "skein"
+version = "0.1.0"
+authors = ["RustCrypto Developers"]
+license = "MIT OR Apache-2.0"
+description = "Skein hash functions"
+repository = "https://github.com/RustCrypto/hashes"
+keywords = ["crypto", "skein", "hash", "digest"]
+categories = ["cryptography", "no-std"]
+edition = "2021"
+rust-version = "1.57"
+
+[dependencies]
+block-buffer = "0.10"
+cipher = "0.4"
```

Comment:
`block-buffer` is re-exported by the `digest` crate, so you may remove it from Cargo.toml. Same with `cipher`, which is re-exported by `threefish`.